app/src/test/java/org/mozilla/focus/search/SearchEngineWriterTest.kt

```diff
@@ -0,0 +1,38 @@
+package org.mozilla.focus.search
+
+import android.graphics.Bitmap
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.robolectric.RobolectricTestRunner
+import org.junit.Assert.assertEquals
+
+@RunWith(RobolectricTestRunner::class)
+class SearchEngineWriterTest {
+
+    @Test
+    fun buildSearchEngineXML() {
+        val endSearchTermXml = "<?xml version=\"1.0\" encoding=\"UTF-8\" standalone=\"no\"?>" +
+            "<OpenSearchDescription xmlns=\"http://a9.com/-/spec/opensearch/1.1/\">" +
+            "<ShortName>testEngine</ShortName>" +
+            "<Image height=\"16\" width=\"16\">data:image/png;base64,Qml0bWFwICgxMCB4IDEwKSBjb21wcmVzc2VkIGFzIFBORyB3aXRoIHF1YWxpdHkgMTAw\n</Image>" +
```

Comment:
\[detekt\] INFO: \[style/MaxLineLength\] Line detected that is longer than the defined maximum line length in the code style\.